src/index.html

```diff
@@ -166,9 +166,6 @@ <h2>Calcite Component Examples - WIP !</h2>
         <li>
           <calcite-link href="/demos/calcite-tabs.html">calcite-tabs</calcite-link>
         </li>
-        <li>
-          <calcite-link href="/demos/calcite-tile.html">calcite-tile</calcite-link>
```

Comment:
Needs to be `calcite-tile-select` not `calcite-tile`.

Reply:
yes, I removed that.

Reply:
Ah, yes I see that now, sorry for the confusion!  I was forgetting this was a removal.